stories/components/color.stories.js

```diff
@@ -0,0 +1,36 @@
+import { h } from 'preact';
+import { Button } from '@Components/Button';
+
+export default {
+  title: 'Design System/Styles/Colors',
+  component: Button,
+  argTypes: {
+    onClick: {
+      action: 'clicked'
+    },
+    variant: {
+      control: {
+        type: 'select',
+        options: [
+          "black",
+          "grey-1",
+          "grey-2",
+          "grey-3",
+          "grey-4",
+          "grey-5",
+          "grey-6",
+          "danger",
+          "success"
```

Comment:
This variant is `black` for me when I run Storybook (I think it doesn't exist)

Reply:
Nice catch, thanks for checking